src/apps/logging/logging.cpp

```diff
@@ -195,6 +196,42 @@ namespace ccfapp
       };
       // SNIPPET_END: log_record_prefix_cert
 
+      auto log_record_anonymous = [this](RequestArgs& args) {
```

Comment:
I think this should be based on `record` rather than `log_record_prefix_cert`. The former uses `json_adapter` to use the standard json-parsing and result-setting logic. The latter uses the raw form as an example, but where we don't need this we should avoid it.